lib/model/folder_summary.go

```diff
@@ -22,7 +22,7 @@ import (
 	"github.com/syncthing/syncthing/lib/util"
 )
 
-const minSummaryInterval = time.Minute
+const maxDurSinceLastEventReq = time.Minute
```

Comment:
`maxDurationSinceLastEventReq` for the sake of verbosity seems reasonable perhaps?